apps/docs/package.json

```diff
@@ -4,14 +4,17 @@
   "private": true,
   "scripts": {
     "start": "next start",
+
```

Comment:
do we really need empty spaces in json files?

Reply:
Not particularly, but it seems to be common practice on the project done by Alexandre and Pat so I relied on that.

ex. https://github.com/gsoft-inc/wl-foundry-cli/blob/main/package.json

Reply:
Great my memory thought it was not using space anyways all good!